lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -2013,11 +2013,13 @@ private void standardUpdateActions(TaskImpl oldTaskImpl, TaskImpl newTaskImpl)
       newTaskImpl1.setBusinessProcessId(oldTaskImpl.getBusinessProcessId());
     }
 
-    // owner can only be changed if task is in state ready
+    // owner can only be changed if task is either in state ready or ready_for_review
     boolean isOwnerChanged = !Objects.equals(newTaskImpl1.getOwner(), oldTaskImpl.getOwner());
-    if (isOwnerChanged && oldTaskImpl.getState() != TaskState.READY) {
+    if (isOwnerChanged
+        && oldTaskImpl.getState() != TaskState.READY
+        && oldTaskImpl.getState() != TaskState.READY_FOR_REVIEW) {
```

Comment:
```suggestion
        && !Set.of(TaskState.READY, TaskState.READY_FOR_REVIEW).contains(oldTaskImpl.getState())) {
```

Reply:
What do u think is more readable?

Reply:
Also thought about that but thought I'd stick to && statement. I think mine might be more readable but yours is less boilerplate and kind of fancier so I will implement the suggestion.